features/frameworks/utest/source/utest_shim.cpp

```diff
@@ -66,7 +66,10 @@ static volatile utest_v1_harness_callback_t minimal_callback;
 static volatile utest_v1_harness_callback_t ticker_callback;
 
 // Timeout object used to control the scheduling of test case callbacks
-Timeout utest_timeout_object;
+static Timeout& utest_timeout_object() {
```

Comment:
@pan- can you provide info on the ARMCC tests that were failing? This could be indicative of a more serious problem. I would like to investigate this regardless of if function local static initialization is used by this patch.